dev/simran0117/mymodule.py

```diff
@@ -0,0 +1,81 @@
+import pandas as pd
+import seaborn as sns
+import matplotlib.pyplot as plt
+import numpy as np
+from sklearn.metrics import classification_report, confusion_matrix,accuracy_score
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import classification_report, confusion_matrix,accuracy_score, f1_score
+from sklearn.svm import SVC
+from sklearn.neighbors import KNeighborsClassifier
+import warnings
+
+
+'''
+Splitting data into test and train, I have tried accuracies for different fractions of test data, 
+i.e. 0.2, 0.1, 0.15
+'''
+
+
+def test_train_data(data_X, data_y):
+    '''
+    dataset=pd.read_csv("C://Users//Simran Verma//Outreachy//PRESC//datasets//winequality.csv")
+    data_X = dataset.drop('recommend',axis=1)
+    data_X = data_X.drop('quality', axis=1)
+    data_y = dataset['recommend']
+    '''
```

Comment:
Was this intentional?

Reply:
Initially when I had not passed the values of data_X and data_y as parameters to a function, I had written this, It's not required anymore so I commented it, instead of removing.
 I'll remove it so that there's no confusion.